server/api/db/sqldb/db.py

```diff
@@ -42,6 +42,7 @@
 import server.api.db.session
 import server.api.utils.helpers
 from mlrun.artifacts.base import fill_artifact_object_hash
+from mlrun.common.types import Operation
```

Comment:
in general - prefer full imports over `from`: see [this section](https://github.com/mlrun/mlrun/blob/development/CONTRIBUTING.md#:~:text=When%20importing%20from%20local%20code%2C%20NEVER%20use%3A%20from%20X%20import%20Y%20instead%20use%3A%20import%20X%20or%20import%20X%20as%20Y.%20For%20external%20packages%2C%20it%27s%20acceptable%20to%20use%20from%20X%20import%20Y%20since%20they%20won%27t%20try%20to%20import%20back%20to%20our%20code.)
```suggestion
import mlrun.common.types
```